rst_files/linear_algebra.rst

```diff
@@ -1109,39 +956,29 @@ The next figure shows two eigenvectors (blue arrows) and their images under :mat
 
 As expected, the image :math:`Av` of each :math:`v` is just a scaled version of the original
 
-
 .. code-block:: julia
-  :class: collapse
+    :class: collapse
 
     A = [1 2
-        2 1]
+         2 1]
     evals, evecs = eigen(A)
-    a1, a2 = evals[1], evals[2]
-    evecs = evecs[:, 1], evecs[:, 2]
-    eig_1 = zeros(2, length(evecs))
-    eig_2 = zeros(2, length(evecs))
-    labels = []
-
-    for i ∈ eachindex(evecs)
-        v = evecs[i]
-        eig_1[2, i] = v[1]
-        eig_2[2, i] = v[2]
-    end
 
+    a1, a2 = evals
+    eig_1 = [0 0; evecs[:,1]']
```

Comment:
Bit terse, but I don't think this is above the students' level. Maybe worth reminding them of what `evecs` returns.

Reply:
There is a markdown cell just below doing exactly that.